src/PresentationalComponents/ZeroState/AppZeroState.js

```diff
@@ -51,72 +45,36 @@ const NewAppZeroState = ({
         return () => {
             mounted.current = false;
         };
-    }, [axios, customFetchResults, hasSystems]);
-
-    return (
-        hasSystems ? children :
-            <IntlProvider>
-                {app.toLowerCase() === 'dashboard'
-                    ? <ZeroState />
-                    :  <Fragment>
-                        <ZeroStateBanner
-                            appName={app}
-                            customInstructions={customInstructions}
-                            customButton={customButton}
-                            customText={customText}
-                            customTitle={customTitle}
-                            appId={appId}
-                        />
-                        {customSection && customSection}
-                        <AppSection appName={app}/>
-                        <ZeroStateFooter appName={app} />
-                    </Fragment>
-                }
-            </IntlProvider>
-    );
-};
+    }, [axios, children, customFetchResults, hasSystems]);
 
-//We will slowly migrate away from this
-const OldAppZeroState = ({
-    app,
-    customInstructions,
-    customButton,
-    customText,
-    customTitle,
-    appId
-}) => {
+    //If there are children, act as a wrapper, otherwise a component
     return (
         <IntlProvider>
-            <React.Fragment>
-                <ZeroStateBanner
-                    appName={app}
-                    customInstructions={customInstructions}
-                    customButton={customButton}
-                    customText={customText}
-                    customTitle={customTitle}
-                    appId={appId}
-                />
-                <AppSection appName={app}/>
-                <ZeroStateFooter appName={app} />
-            </React.Fragment>
+            {(children && hasSystems) ? (
+
+                children
+            ) : (
+                <>
+                    <ZeroStateBanner
+                        appName={app}
+                        customInstructions={customInstructions}
+                        customButton={customButton}
+                        customText={customText}
+                        customTitle={customTitle}
+                        appId={appId}
+                    />
+                    {customSection && customSection}
+                    <AppSection appName={app} />
+                    <ZeroStateFooter appName={app} />
+                </>
+            )}
         </IntlProvider>
     );
 };
 
 export default AppZeroState;
 
-OldAppZeroState.propTypes = {
-    app: propTypes.string,
-    customInstructions: propTypes.any,
-    customButton: propTypes.any,
-    customText: propTypes.string,
-    customTitle: propTypes.string,
-    appId: propTypes.string,
-    children: propTypes.any,
-    fetchSystem: propTypes.bool
-};
-
-NewAppZeroState.propTypes = {
+AppZeroState.propTypes = {
```

Comment:
You are shadowing propTypes, line 88 has the same AppZeroState referenced.
You should likely combine the two. :)

Reply:
ahh good catch!